stats/models/match_history.py

```diff
@@ -17,7 +17,9 @@ def retrieve_matches(self):
         matches_ids = self.riotProxy.get_matches(self.person.puuid)
         for match_id in matches_ids:
             match_data = self.riotProxy.get_match(match_id)
-            if match_data["info"]["gameMode"] == "ARAM":
+            if 'status' in match_data:
```

Comment:
I had to google the purpose of a django model.  From what I found, a django model is: 

"the built-in feature that Django uses to create tables, their fields, and various constraints. In short, Django Models is the SQL Database one uses with Django."

Seems like this should be nothing more than a data contract, not a service.  Why do you have proxy calls here?

Reply:
Absolutely agree

Reply:
Will add todo to make models just be DTO/DBO while that functionality exists somewhere else